src/menus/block.ts

```diff
@@ -107,6 +107,12 @@ export default function blockMenuItems(
       icon: ImageIcon,
       keywords: "picture photo",
     },
+    {
+      name: "file",
+      title: "File",
+      icon: ImageIcon,
+      keywords: "file doc pdf",
```

Comment:
```suggestion
      keywords: "doc pdf",
```

No need to include "file" as the name and title are included in the keywords automatically

Reply:
Okay got it